Header header
float32 SensorAngleBoom
float32 SensorAngleArm
float32 SensorAngleBucket
